yolo_object_detection/DetectedObject[] box